Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -8609,48 +8613,139 @@ def get_message_classes_dump(classes_as_list):
 
 def get_message_local_vars():
     """
-    A function that prints the local variables to log info
+    A function that returns the printable message about local variables
 
     :return: Message to print.
     :rtype: ``str``
     """
     local_vars = list(locals().items())
     ret_value = '\n\n--- Start Local Vars ---\n\n'
     for current_local_var in local_vars:
-        ret_value += str(current_local_var) + '\n'
+        ret_value += shorten_string_for_printing(str(current_local_var), 64) + '\n'
 
     ret_value += '\n--- End Local Vars ---\n\n'
 
     return ret_value
 
 
+def get_size_of_object(input_object):
+    """
+    A function that recursively iterate to sum size of object & members.
+
+    :type input_object: ``Any``
+    :param input_object: The object to calculate its memory footprint
+
+    :return: Size of input_object in bytes.
+    :rtype: ``int``
+    """
+    from collections import deque, Mapping
+    from numbers import Number
+    # Python 3 also needs range
+    # ZERO_DEPTH_BASES = (str, bytes, Number, range, bytearray)
```

Comment:
Is this intended to be an if for `py2` vs `py3`? if so why is this commented?

Reply:
It's a reminder that when we port CommonServerPython to py3 then we should add the `range`.

Reply:
Where does this happen? Perhaps just use the `IS_PY3` flag and adjust the behavior?

Reply:
https://github.com/demisto/content/pull/16776/commits/1e8a0b08b89c3369c9bc0b1160bb390957bdc1fb